src/main/java/net/finmath/time/TimeDiscretizationInterface.java

```diff
@@ -90,4 +90,14 @@
 	 * @return A new time discretization where all time points have been shifted by the given time shift.
 	 */
 	TimeDiscretizationInterface getTimeShiftedTimeDiscretization(double timeShift);
+
+	/**
+	 * @param that Another time discretization containing points to add to the time discretization.
```

Comment:
If JavaDoc is present a description should be present. The behaviour of tickSize within the union and intersection should be documented. To be honest: The tickSize feature (there was a use case for this) makes me wonder if union and interception are well defined operators. For example we have hat for two disjoint sets A and B (A union B) interesect B is not equal to B.  - not sure here. API has to be clarified (max/min on tickSize?!)